scene/gui/color_picker.cpp

```diff
@@ -54,6 +54,11 @@
 #include "scene/theme/theme_db.h"
 #include "thirdparty/misc/ok_color_shader.h"
 
+#ifdef TOOLS_ENABLED
+#include "editor/editor_node.h"
+#include "editor/gui/editor_quick_open_dialog.h"
+#endif // TOOLS_ENABLED
```

Comment:
You can't add `editor/` includes to `scene/`. #53295

Reply:
Currently, `ColorPicker` is using type erasure to avoid including `editor/` while it's still using editor features. I think this kind of situation is self deception, and it makes tracking such situation a lot harder.

https://github.com/godotengine/godot/blob/d7382aa05854df37c2f21aff02a693f8104fb95d/scene/gui/color_picker.cpp#L610

I'll find a proper way to solve this.